src/stores/sagas/customerWeekDaysSaga.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  all, call, put, takeLatest,
+} from 'redux-saga/effects';
+
+import { ICustomerWeekDaysType } from 'models/customerWeekDays/ICustomerWeekDaysType';
+import {
+  getCustomerWeekDaysFailure,
+  getCustomerWeekDaysSuccess,
+  postCustomerWeekDaysFailure,
+  postCustomerWeekDaysSuccess,
+} from 'stores/actions/customerWeekDays/customerWeekDaysActions';
+import CustomerWeekDaysActionTypes from 'stores/actions/customerWeekDays/customerWeekDaysTypes';
+import { sessionToken } from 'stores/reducers/tokenReducer';
+import refreshAccessToken from 'stores/sagas/utils';
+import apiClient, { getRequest } from 'utils/apiClient';
+
+export const customerWeekDaysUrl = 'customer_week_days/';
+
+export const postRequest = async (payload: object, id: number) => {
```

Comment:
@KarolinaKopacz in the future we could add this into 'apiClient.tsx'. But you don't have to do this now.